packages/lesswrong/components/posts/PostsHighlight.tsx

```diff
@@ -46,13 +47,13 @@ const PostsHighlight = ({post, maxLengthWords, classes}: {
       rawWordCount={wordCount}
       expanded={expanded}
       getTruncatedSuffix={({wordsLeft}: {wordsLeft:number}) => <div className={classes.highlightContinue}>
-        {wordsLeft > 1000
-          ? <Link to={postGetPageUrl(post)}>
-              (Continue Reading – {wordsLeft} more words)
-            </Link>
-          : <Link to={postGetPageUrl(post)} onClick={clickExpand}>
+        {(forceSeeMore || wordsLeft > 1000)
+          ? <Link to={postGetPageUrl(post)} onClick={clickExpand}>
```

Comment:
For Recent Discussion, this swapped which case expands the post and which case links, so that would should be a small expand is a link, and what should be a link is a giant expand.

Reply:
Seems fine for me to take this on since it was my mistake.